config_test.go

```diff
@@ -375,7 +375,7 @@ func checkDefault(t *testing.T, c *hazelcast.Config) {
 	assert.Equal(t, "", c.Cluster.Cloud.Token)
 
 	assert.Equal(t, cluster.ReconnectModeOn, c.Cluster.ConnectionStrategy.ReconnectMode)
-	assert.Equal(t, types.Duration(9223372036854775807), c.Cluster.ConnectionStrategy.Timeout)
+	assert.Equal(t, types.Duration(31622400000000000), c.Cluster.ConnectionStrategy.Timeout)
```

Comment:
What is this value ? Can we make it more obvious so that we understand what it is ?